Library/Tests/TestHelpers/MockBundle.swift

```diff
@@ -54,17 +54,17 @@ internal struct MockBundle: NSBundleType {
     self.store = stores[lang] ?? [:]
   }
 
-  internal static func create(path path: String) -> NSBundleType? {
+  internal static func create(path: String) -> NSBundleType? {
     return MockBundle(lang: path)
   }
 
-  internal func localizedStringForKey(key: String, value: String?, table tableName: String?) -> String {
+  internal func localizedString(forKey key: String, value: String?, table tableName: String?) -> String {
     // A real `NSBundle` will return the key if the key is missing and value is `nil`.
-    return self.store[key] ?? value ?? key ?? ""
+    return self.store[key] ?? value ?? key
```

Comment:
swift 3 now has a warning when you try to use `??` on a non-optional value, and it found quite a few we were accidentally doing!